### DISCUSSION TITLE
calltime variable : declare and init variable by calling

### DISCUSSION THREAD

**Discussion by nutix56** [dotnet/csharplang#4112] [Language Ideas]
`typeof(Form).GetMembers()
> .Select(m => $"{m.Name} => {m.MemberType}") 
> .ToJoinedText("\r\n")
> **.SaveAs("Form.txt" _: path_);
O.RunFile(_path_);**`

see the code

we can program no-break with that.

name : value => named parameter
value : name => calltime variable

**Comment by HaloFour** [comment]
What is `path` supposed to be here?  What is its relationship to the argument "Form.txt"?

**Comment by yaakov-h** [comment]
Is this just another form of declaration expressions (#973)? I don't _quite_ understand the provided sample code or what it's supposed to translate into.

**Comment by theunrepentantgeek** [comment]
What problem does this solve?

A random piece of code with no narrative or motivation is _*very*_ difficult to decipher.
